src/components/DayPicker.jsx

```diff
@@ -825,6 +825,9 @@ export default class DayPicker extends React.Component {
       <div
         className={dayPickerClassNames}
         style={dayPickerStyle}
+        role="application"
+        aria-label="Calendar"
+        aria-describedby="CalendarMonth__caption"
```

Comment:
I'm curious if this could be confusing - since it will be most likely just read when the user comes to the calendar the first time, the description will always be the first month. They will come to an area that will read something like, "Calendar, application, September". Seems like a more appropriate description would be something like "Date picker"

Reply:
Hmm well, going back and testing this again to be sure, it seems as if it's not even reading the aria-describedby.  I must have assumed that when I heard "September 2017" it was the aria-describedby but really it was just the reading of the selected CalendarDay.  Upon changing the text it's now more obvious that it's not being read.  Perhaps it's just not supported by the browser screen reader combinations I'm testing on (Mac with VoiceOver on Chrome, Firefox, Safari)?  I so far can't find a good reason for why it's not being read... 

I wonder if it's really necessary or if the aria-label here and the aria-describedby on the input telling the user how to interact with the calendar is enough information?  If you think a describedby is crucial here let me know and I'll keep working on why it's not being read, otherwise I can just remove it.

Reply:
@majapw and @backwardok, I just wanted to bump this and ask again, what are your thoughts about whether to skip adding an aria-describedby on the role="application" div or to figure out why it's not being read and fix it.  If the answer is to add it and fix it, what should the text be?

Reply:
Sorry for the delay! I think it would be fine to bypass adding the aria-describedby here. If we were to have some sort of visual text that also explained the calendar, or there was text that explained some calendar instructions specifically for screen reader users, then that might be useful, but I think it's probably ok without it.

Reply:
No worries!  I have added the aria-label text to the localizable phrases and removed the aria-describedby.  

As for having something visual and read out loud explaining the calendar instructions, there already is the keyboard shortcuts modal which is opened by entering ? or clicking on the ? at the bottom of the calendar.  The instructions for getting to the keyboard shortcuts modal is read via the aria-describedby on the DateInput:

`Press the down arrow key to interact with the calendar and
  select a date. Press the question mark key to get the keyboard shortcuts for changing dates.`

So the instructions seem pretty clear and complete from my testing as a visually able, keyboard only, and screen-reader user.